app/src/main/java/pl/droidevs/books/library/BookActivity.java

```diff
@@ -79,80 +75,82 @@ protected void onCreate(Bundle savedInstanceState) {
         ButterKnife.bind(this);
 
         if (savedInstanceState != null) {
-            bookId = savedInstanceState.getString(EXTRAS_BOOK_ID);
-            imageTransitionName = savedInstanceState.getString(EXTRAS_IMAGE_TRANSITION_NAME);
-            titleTransitionName = savedInstanceState.getString(EXTRAS_TITLE_TRANSITION_NAME);
-            authorTransitionName = savedInstanceState.getString(EXTRAS_AUTHOR_TRANSITION_NAME);
-            shadowTransitionName = savedInstanceState.getString(EXTRAS_SHADOW_TRANSITION_NAME);
-            lastSelectedIndex = savedInstanceState.getInt(EXTRAS_LAST_SELECTED_INDEX);
+            this.animationBundle = savedInstanceState.getBundle(BUNDLE_EXTRAS);
```

Comment:
ustawiam bundle pobrane z saveedInstance na animationBundle

jako że przechowuję jako bundle to jest mi prościej całe bundle potem wysłać niż wyciągać z niego. Zatem zawsze przy odczycie  trzeba teraz całe bundle pobrać najpierw